Gems/StartingPointCamera/Code/Source/CameraLookAtBehaviors/SlideAlongAxisBasedOnAngle.h

```diff
@@ -23,8 +23,7 @@ namespace Camera
     /// will occur when looking up.  This could also be used for peeking around
     /// corners.  This is primarily useful for third person cameras.
     //////////////////////////////////////////////////////////////////////////
-    class SlideAlongAxisBasedOnAngle
-        : public ICameraLookAtBehavior
+    class SlideAlongAxisBasedOnAngle : public ICameraLookAtBehavior
```

Comment:
I believe our coding standard is for the public to be on a newline like it was before, unless that has changed?

```
class SlideAlongAxisBasedOnAngle
        : public ICameraLookAtBehavior
```

Reply:
`clang-format` will stack them when there's more than one but puts them on the same line when there's only one, I think this is something worth living with to get auto-formatting so I'd say it's okay to leave as-is